examples/fastai/conda.yaml

```
@@ -0,0 +1,11 @@
+channels:
+- defaults
+- anaconda
+- conda-forge
+dependencies:
+- python=3.7.5
+- pip
+- pip:
+  - mlflow
```

Only versions of mlflow with fastai support will be able to use this.
```suggestion
  - mlflow>=1.x.x
```

---

Normally, it gets the latest one but good to point out that it was first introduced in 1.9.0 (#2619, #2689)

---

done!